pkg/config/config.go

```diff
@@ -84,6 +86,7 @@ func setDefaults(v *viper.Viper) {
 	v.SetDefault("database.name", "")
 	v.SetDefault("certs.cert_path", "")
 	v.SetDefault("options.paged_rpm_inserts_limit", DefaultPagedRpmInsertsLimit)
+	v.SetDefault("options.local", false)
```

Comment:
what does this setting do?

Reply:
I thought about this as a mechanism to disable the middleware with the configuration, but it is not necessary! I have to remove it! thanks!